core/lib/src/form/from_form_field.rs

```diff
@@ -226,11 +226,11 @@ use crate::form::prelude::*;
 #[crate::async_trait]
 pub trait FromFormField<'v>: Send + Sized {
     fn from_value(field: ValueField<'v>) -> Result<'v, Self> {
-        Err(field.unexpected())?
+        Err(field.unexpected().into())
```

Comment:
I think this and the rest in this file are fine. Let's revert them all.

Reply:
I've notice that you prefer the question mark way I've allowed the lint [try_err](https://rust-lang.github.io/rust-clippy/master/index.html#try_err) for each traits with the line :
`#[allow(clippy::try_err)] // Returning Err(...)? is more readable than Err(...into())`
Feel free to allow the lint for the whole project, but as always, I would recommend respecting Clippy's way to get a standard style between projects.

Reply:
I believe my preference is: if it is the return expression of the function, `Err(e)?` is okay. Otherwise if it's an early return, prefer `return Err(e.into())`.

Reply:
Thank you, I'll allow those cases with the comment explaining your preference :
```
#[allow(clippy::try_err)] // For return values, please use Err(...)? instead of .into().
```
I think it will help contributors to maintain that rule.

Reply:
Thank you, I'll allow those cases with the comment explaining your preference :
```
#[allow(clippy::try_err)] // For return values, use : "Err(...)?" For early return, use : "return Err(...into());".
```
I think writing that rule will help new contributors to respect it.